betterproto/tests/test_inputs.py

```diff
@@ -25,7 +25,7 @@ class TestCases:
     def __init__(self, path, services: Set[str], xfail: Set[str]):
         _all = set(get_directories(path))
         _services = services
-        _messages = _all - services
+        _messages = (_all - services) - {"__pycache__"}
```

Comment:
this `{"__pycache__"}` should go on 26 i think

`_all = set(get_directories(path)) - {"__pycache__"}`

Reply:
True